bundles/org.openhab.binding.ftpupload/src/main/java/org/openhab/binding/ftpupload/internal/FtpUploadHandlerFactory.java

```diff
@@ -116,9 +118,21 @@ protected synchronized void modified(ComponentContext componentContext) {
             }
         }
 
+        if (properties.get("passivePorts") != null) {
+            String strPassivePorts = properties.get("passivePorts").toString();
+            if (StringUtils.isNotEmpty(strPassivePorts)) {
```

Comment:
Can you change this to use standard Java. We're trying to get ride of apache libraries, and even while it's used on other places here, now new usages should be added.
```suggestion
            if (!strPassivePorts.isEmpty()) {
```